wp-admin/css/colors/_mixins.scss

```diff
@@ -4,31 +4,31 @@
  */
 @mixin button( $button-color, $text-color: white ) {
 	background: $button-color;
-	border-color: darken( $button-color, 10% );
+	border-color: darken( $button-color, 10% ) darken( $button-color, 15% ) darken( $button-color, 15% );
 	color: $text-color;
-	box-shadow: inset 0 1px 0 lighten( $button-color, 15% ), 0 1px 0 rgba(0,0,0,.15);
+	box-shadow: 0 1px 0 darken( $button-color, 15% );
+	text-shadow: 0 -1px 1px darken( $button-color, 15% ),
+		1px 0 1px darken( $button-color, 15% ),
+		0 1px 1px darken( $button-color, 15% ),
+		-1px 0 1px darken( $button-color, 15% );
 
 	&:hover,
 	&:focus {
-		background: darken( $button-color, 5% );
+		background: lighten( $button-color, 3% );
 		border-color: darken( $button-color, 15% );
 		color: $text-color;
-		box-shadow: inset 0 1px 0 lighten( $button-color, 10% );
+		box-shadow: 0 1px 0 darken( $button-color, 15% );
```

Comment:
Expected 0 spaces after `(` instead of ` `<br>Expected 0 spaces before `)` instead of ` `